subprojects/dependency-management/src/main/java/org/gradle/api/internal/catalog/LibrariesSourceGenerator.java

```diff
@@ -204,10 +207,30 @@ private void writeBundleAccessorClass(ClassNode classNode) throws IOException {
         writeLn();
     }
 
+    private final Map<String, Integer> classNameCounter = new HashMap<>();
```

Comment:
Field declarations should be before the methods.

Reply:
Thanks for the review, updated